packages/flutter/test/widgets/container_test.dart

```diff
@@ -403,6 +408,7 @@ void main() {
         '         │ textDirection: null\n'
         '         │\n'
         '         └─child: RenderPositionedBox#00000\n'
+        '           │ needsCompositing: false\n'
```

Comment:
We shouldn't display this flag when the value is false. For example, typical inspector users should care less that `needsCompositing: false` as generally the inspector should only be querying Data after a frame has completed so it will always be false.

Reply:
I'm a little confused why the code you wrote would be indicating that this flag is false. Hopefully you generated these updated goldens before tweaking your code otherwise there is a bug in FlagProperty which we should fix.

Reply:
Yeah, I agree.  I was surprised to see that it printed the name instead of the "needs compositing" string. I assumed it had something to do with the treatment of the `hidden` log level. These are what got generated after I made my changes.

Reply:
Ah ok. The output makes complete sense now.